tests/tests_lite/accelerators/test_cuda.py

```diff
@@ -0,0 +1,73 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
A mix of test cases carried over from tests/pytorch/accelerators/test_gpu.py and newly written ones